_posts/2015-11-01-practicing-well-crafted-design.md

```diff
@@ -15,10 +15,10 @@ tags:
 - deliberate practice
 ---
 
-Katas are great, but not enough. In the past year I’ve been trying to find ways to teach our craftsmen and apprentices how to design software. Although we could discuss good design principles, it was very difficult to find a way to practice them. Katas are great for learning TDD, micro design, working in baby steps, and a few techniques for dealing with legacy code but we need more. We would like to practice requirements gathering, domain modelling, and well-crafted code all together, with the same fast feedback loop we get when we do katas. The problem is that we need a far bigger problem with overlapping rules so that we can experience incremental software design. Inspired by a session we had a [Socrates Germany 2015][6], we are evolving a deliberate practice session that is allowing us to achieve that.
+Katas are great, but not enough. In the past year I’ve been trying to find ways to teach our craftsmen and apprentices how to design software. Although we could discuss good design principles, it was very difficult to find a way to practice them. Katas are great for learning TDD, micro design, working in baby steps, and a few techniques for dealing with legacy code but we need more. We would like to practice requirements gathering, domain modelling, and well-crafted code all together, with the same fast feedback loop we get when we do katas. The problem is that we need a far bigger problem with overlapping rules so that we can experience incremental software design. Inspired by a session we had a [SoCraTes Germany 2015][6], we are evolving a deliberate practice session that allows us to achieve that.
 
 # The Session 
-The objective is to pick a game (card, board, dice) and implement the code in a way where the code clearly describes the rules of the game. Any person that was not part of the group that wrote the code should be able to look at the code and very quickly understand how the game works.
+The objective is to pick a game (card, board, dice) and implement the code in a way where the code clearly describes the rules of the game. Any person not part of the group that wrote the code should be able to look at the code and very quickly understand how the game works.
 
 ### Setup
 The session is run as a mob programming. Make sure you have a projector and a whiteboard or flip chart.
```

Comment:
While we're at it, this should probably say "run as a mob-programming session" or "run using mob programming".